lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/AWSParamStoreRetriever.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.di.ipv.cri.kbv.api.service;
+
+import software.amazon.lambda.powertools.parameters.ParamManager;
+import software.amazon.lambda.powertools.parameters.SSMProvider;
+import uk.gov.di.ipv.cri.kbv.api.config.ConfigurationConstants;
+
+import java.util.Objects;
+
+public class AWSParamStoreRetriever {
+    private final SSMProvider ssmProvider;
+    private final String parameterPrefix;
+
+    public AWSParamStoreRetriever() {
```

Comment:
This class can be removed as the SSM parameter retrieval is handled by the ConfigurationService. ( See other comment on the ClockService class )